.gitignore

```diff
@@ -21,3 +21,11 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# ignore type definitions
+*.d.ts
+*.map
+
+# ignore produced files
+lib/*
```

Comment:
Please remove `lib` and `docs`  folder 
from ignore as we are using lib and docs as github.io static host

Reply:
These produce lots of noise on git status, and should part of the release's DevOps process.
However you are in control of the release - so removed them.